lib/kubernetes-deploy.rb

```diff
@@ -34,10 +34,14 @@ class FatalDeploymentError < StandardError; end
     Pod
   )
 
+  def self.logger=(value)
+    @logger = value
+  end
+
   def self.logger
     @logger ||= begin
-      l = Logger.new(STDOUT)
-      l.level = ENV["DEBUG"] ? Logger::DEBUG : Logger::INFO
+      l = Logger.new($stderr)
```

Comment:
as mentioned by @sirupsen in https://github.com/Shopify/kubernetes-deploy/pull/1/files#r96518542